views/assets/javascripts/tyk.js

```diff
@@ -1,3 +1,18 @@
+$(document).ready(function () {
```

Comment:
@padiazg @ConsM this should be built like a proper Javascript plugin functionality, and not threw in a Document ready callback.

the format of this could look smth like this:

```
var Collapsable = function(options) {

    cons publicMethods = {};

   plubicMethods.collapse = () => {};
   plubicMethods.expand = () => {};
    
   return publicMethods;
};

let myCollapsable = Collapsable({selector: '.collapsable'}); //you can pass other options if needed

```

Reply:
Also I would suggest to create functionality for each plugin in their own file so we don't pollute tyk.js, this should be kept like the entry point of the JS code, the place where things get intiialized

Reply:
I would also recommend not to do it with JQuery since there are no heavy stuff needed to be done with it. Plain JS can do all the stuff nicely.